virtual-desktop/src/app/plugin-manager/plugin-factory/angular2/angular2-plugin-factory.ts

```diff
@@ -81,7 +81,7 @@ export class Angular2PluginFactory extends PluginFactory {
     let pluginDefBase = pluginDefinition.getBasePlugin();
     let pluginDefAny:any = (pluginDefBase as any);
     let entryPoint = 'main.js';
-    if (pluginDefAny.getWebEntryPoint) {
+    if (pluginDefAny.getWebEntryPoint()) {
```

Comment:
this changes the meaning from "does this exist" to "is its return null"
you will throw an exception with this.
the purpose of the if() is to check if the function exists, since it is not guaranteed to.